package.json

```diff
@@ -44,6 +44,7 @@
     "eslint-plugin-jsx-a11y": "^6.0.3",
     "eslint-plugin-prettier": "^2.6.0",
     "eslint-plugin-react": "^7.6.1",
+    "eslint_d": "^5.2.2",
```

Comment:
How come you have added this dependency?

Reply:
I've rebased onto master as it was behind, this is something Hugo added to have `eslint` working with his setup

Reply:
👍

Reply:
Erm actually, a rebase means that it shouldn't appear in this PR

Reply:
You're right now that I'm thinking of it 😱